CliClient/locales/zh_CN.po

```diff
@@ -574,11 +575,11 @@ msgstr "此答案无效：%s"
 
 #: CliClient/app/cli-utils.js:160 CliClient/app/app.js:136
 msgid "Y"
-msgstr "是"
+msgstr "是 (Y)"
 
 #: CliClient/app/cli-utils.js:160 CliClient/app/app.js:136
 msgid "n"
-msgstr "否"
+msgstr "否 (n)"
```

Comment:
Have you tried this? This might not work as expected. As far as I know the code only expects a single character.

Reply:
I‘m sorry for that again. Because some lines have not been translated before, and I didn't see much of the original text during my using the app ...... I will take the time to improve it again. Sorry to make myself an idiot.

Reply:
No, as I said, I haven't tested it myself, but I recall that the app checks for a single character in the answer. I don't know what happens, if there are more than one character.
Just to be safe, let's just use one char. If you've tested it, and it works with more characters, we can change it in the future...